app/main.py

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
please remove this empty lines

Reply:
Without these lines I have no validation on the flake8